backend/package.json

```diff
@@ -42,7 +48,8 @@
     "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-import": "^2.20.1",
     "jest": "^25.1.0",
-    "prettier": "^1.19.1",
+    "prettier": "^2.0.5",
+    "sqlite3": "^4.2.0",
```

Comment:
Is the sqlite necessary anymore if we could use postgres's image for development as well?